README.md

```diff
@@ -15,7 +15,6 @@ The following Repos will be setup and enabled by default:
 Other repositories that will setup but disabled by default:
 
   * atomic-testing
-  * atomic-bleeding
```

Comment:
@andyshinn It no longer exists as far as I can tell.  `http://updates.atomicorp.com/channels/mirrorlist/atomic-bleeding/centos-6-x86_64` does not work.